src/Button/Button.js

```diff
@@ -1,54 +1,109 @@
-import React from 'react'
-import propTypes from 'prop-types'
+import React from 'react';
+import PropTypes from 'prop-types';
+import { buttonVariants, buttonSizes, buttonTypes } from './Button.levers';
 
 /**
- * Primary UI component for user interaction
+ * Button Component
+ *
+ * HTML button element
  */
-export const Button = ({ className, children, onClick, type, ref, ...props }) => {
+export const Button = ({ className, children, onClick, ref, variant, size, type, isDisabled, ...props }) => {
   // Defaults & Variables.
   // ---------------------------------------------------------------------------
-  const classnames = require('classnames')
-  const levers = {}
+  const classnames = require('classnames');
+  const levers = {};
 
   // Levers
   // ---------------------------------------------------------------------------
 
+  // Props.variant & if button not disabled
+  if (variant && buttonVariants.includes(variant) && !isDisabled) {
```

Comment:
```suggestion
  // Props.variant 
  if (variant && buttonVariants.includes(variant)) {
```

I don't think we need to check for disabled. You can have a disabled secondary button no?

Reply:
I'm going to ask Kerri if we can have all the disabled button with the same style since the design for the disabled secondary is not accessible.

Reply:
Proposed some changes in a new PR. Check it out.